tests/test_mtx.py

```diff
@@ -93,6 +93,8 @@ def test_get_data_start_line_number(self):
         mtx_file_handler = open("data/mtx/unsorted_mtx.mtx.txt")
         self.assertEqual(3, MTXIngestor.get_data_start_line_number(mtx_file_handler))
 
+        mtx_file_handler = open("data/mtx/mtx_character_before_data.mtx")
+        self.assertEqual(5, MTXIngestor.get_data_start_line_number(mtx_file_handler))
```

Comment:
Processing mtx_character_before_data.mtx should result in a ValueError because the MTX file does not conform to the file format spec. A separate test file with extra comment lines (ie. prefixed with the % symbol) with "a" or "/" as values should be used for this particular test where we expect that the new start line is 5.